Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.py

```diff
@@ -76,9 +76,12 @@ def login():
     response = session.get(login_url, verify=VERIFY_SSL)
 
     # get the VIEW_STATE from the xml returned in the UI login page.
+    resp_text = response.text.encode('utf-8')
```

Comment:
I think now is a good opportunity to mock the response for future uses.
make sure it's secret-free before posting it.

Reply:
also, if the login page was changed - this will have to be re-written as this request is not part of the REST API.